ebu_tt_live/xsd/ebutt_metadata.xsd

```diff
@@ -15,6 +15,7 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
     xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning" vc:minVersion="1.1">
 	<xs:import namespace="urn:ebu:tt:datatypes" schemaLocation="ebutt_datatypes.xsd"/>
 	<xs:import namespace="http://www.w3.org/ns/ttml#metadata" schemaLocation="metadata.xsd"/>
+	<xs:attribute name="role" type="xs:NMTOKENS"/>
```

Comment:
Won't this define it in the `urn:ebu:tt:metadata` namespace? I'm not sure we need the line at all - it's already defined at https://github.com/bbc/ebu-tt-live-toolkit/blob/85cae07ace7f401b53cb9a4b17d9ade643d87be5/ebu_tt_live/xsd/metadata.xsd#L13 which we import in the line above.

Reply:
removed